src/notificationsWidget/manageTeamWebview.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  commands,
+  ExtensionContext,
+  WebviewView,
+  WebviewViewProvider,
+  window,
+} from "vscode";
+import layout from "../utils/webviewLayout";
+import { getHubBaseUrl } from "../utils/binary.utils";
+import { Capability, isCapabilityEnabled } from "../capabilities/capabilities";
+import { fireEvent } from "../binary/requests/requests";
+
+function registerNotificaitonsWebviewProvider(context: ExtensionContext): void {
+  const provider = new NotificationsWebviewProvider();
+  void setManageTeamWebviewReady();
+
+  context.subscriptions.push(
+    window.registerWebviewViewProvider("tabnine-notifications", provider)
+  );
+}
+
+function setManageTeamWebviewReady() {
+  if (isCapabilityEnabled(Capability.NOTIFICATIONS_WIDGET)) {
```

Comment:
isCapabilityEnabled can throw an exception, and u r void(ing) on `setManageTeamWebviewReady` call

Reply:
It cant throw exception - but I changed the order and remove the void